tests/dummy/app/templates/application.hbs

```diff
@@ -6,4 +6,9 @@
   </div>
 {{/code-block}}
 
+<h3>With `line-numbers` plugin:</h3>
+{{#code-block class="line-numbers"}}&lt;div&gt;
```

Comment:
I think since the original Prism detects class for the `line-numbers` plugin, it is a natural thing to follow what they are doing. So that when newcomers use this addon they don't need to look up for the documentation for an addon-specific way to use the plugin.

That being said, Prism is quite inconsistent in their plugin syntax, it may be a good thing to unify them in the future, something like:

```
{{#code-block plugins="line-numbers, show-language, autolinker" template="coy"}}
...
{{/code-block}}
```

It would be way cleaner, and can have separated plugins/templates for different `code-blocks`. Though it would also be a hell of a task to work on!